### ISSUE TITLE
Warning with PHP 8.0.27

### ISSUE THREAD

**Issue by michaelbeil** [strangerstudios/pmpro-advanced-levels-shortcode#48]
**Describe the bug**
A customer ran into a warning with PHP 8.0.27.

**To Reproduce**
Steps to reproduce the behavior for customer:
1. Set PHP version to PHP 8.0.27
2. Activate Paid Memberships Pro, and Advanced Levels Page Shortcode Add On 
3. Go to the the levels page. See this error:

```
Warning: Undefined property: stdClass::$allow_signups in /xxyy/public_html/wp-content/plugins/pmpro-advanced-levels-shortcode/templates/levels-table.php on line 91
```

**Expected behavior**
There should not be an error.

This came up in ticket 482012 on PHP 8.0.27, but I couldn't replicate on PHP 8.0.22.

**WordPress Environment**
<details> 
Paid Memberships Pro 2.10.3, and Advanced Levels Page Shortcode Add On 0.2.5, WordPress 6.1.1, PHP 8.0.27
</details>

**Comment by dwanjuki** [comment]
Another customer facing this issue in ticket 486916. It causes the _Renew Button_ to not show as expected for expiring levels on the _table_ and _compare_table_ layout. I was able to reproduce on PHP 8.0.28

**Comment by aquiferweb** [comment]
I encountered this when using the advanced levels shortcode today - and I can confirm that @JarrydLong 's fix #50 appears to resolve this.
